### PR TITLE
chore: [40] initialize readme

### PR DESCRIPTION
**Pull request by mmnoo**

### CODE REVIEW

**Comment by mmnoo** [comment]
John will add the section on the product.

File: mrtt-ui/README.md [hunk 1]
```diff
@@ -1,50 +1,49 @@
...
+- Mapbox to draw mangrove site locations.
+- [The Global Mangrove Watch API](https://github.com/globalmangrovewatch/gmw-api) to handle data on the server side
+- Transifex for user language translation.
 
-### `npm test`
+## Archetectural Decisions and Tradeoffs
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- Translation will be provided via Transifex which is configured at `mrtt-ui/public/index.html`. This is unlikely to translate dynamically generated text, so we included a language file (`mrtt-ui/src/language.js`)for easy future translation. We attempted to put all user facing text in the language file, but in some places it is hardcoded in the JSX
```
Comment (julia-rom) [inline mrtt-ui/README.md:26]: nice job adding this note!

File: mrtt-ui/README.md [hunk 2]
```diff
@@ -1,50 +1,49 @@
...
+- Theming is configured in two files.
+  - `mrtt-ui/src/styles/themeMui.js` is responsible for setting style defaults for MUI components
+  - `mrtt-ui/src/styles/theme.js` is for all other style tokens. `themeMui.js` should import tokens from `theme.js`
+- This project had a very tight timeline, so it was decided that we skip writing front end tests.
+- The various site data items/form questions and answers have numerical ids based on question order on the server side. It was decided that the front end would use descriptive naming for coding these questions and answers. We map the descriptive names to the ids used on the server in the `mrtt-ui/src/library/mapDataForApi.js` file using a a lookup object found here: `mrtt-ui/src/data/questionMapping.js`.
```
Comment (julia-rom) [inline mrtt-ui/README.md:31]: maybe worth mentioning [questionMapping.j](url)s too

**Review by julia-rom — APPROVED** [review]

File: mrtt-ui/README.md (hunk 1 above)
Comment (mmnoo) [reply mrtt-ui/README.md:26]: I only remembered cause you mentioned it, lol

File: mrtt-ui/README.md (hunk 2 above)
Comment (mmnoo) [reply mrtt-ui/README.md:31]: i mention it when I refer to the 'look up object'? Are you thinking of adding more details about it?

**Pull request merged**